### PR TITLE
Slight Code Refactoring

### PR DESCRIPTION
**Pull request by ACGNnsj**
🙂I've done some refactoring aming at better readability.

### CODE REVIEW

File: Ryujinx.Horizon.Kernel.Generators/Kernel/SyscallGenerator.cs [hunk 1]
```diff
@@ -151,24 +152,9 @@ public void Execute(GeneratorExecutionContext context)
...
-                        foreach (var attributeArg in attribute.ArgumentList.Arguments)
-                        {
-                            if (attributeArg.Expression.Kind() == SyntaxKind.NumericLiteralExpression)
-                            {
-                                LiteralExpressionSyntax numericLiteral = (LiteralExpressionSyntax)attributeArg.Expression;
-                                syscalls.Add(new SyscallIdAndName((int)numericLiteral.Token.Value, method.Identifier.Text));
-                            }
-                        }
-                    }
+                    syscalls.AddRange(from attributeArg in attribute.ArgumentList.Arguments where attributeArg.Expression.Kind() == SyntaxKind.NumericLiteralExpression select (LiteralExpressionSyntax)attributeArg.Expression into numericLiteral select new SyscallIdAndName((int)numericLiteral.Token.Value, method.Identifier.Text));
```
Comment (gdkchan) [inline Ryujinx.Horizon.Kernel.Generators/Kernel/SyscallGenerator.cs:157]: I don't mind LINQ, but those extremely long lines are not good for readability imo. Maybe you can put some line breaks here.

Comment (ACGNnsj) [reply Ryujinx.Horizon.Kernel.Generators/Kernel/SyscallGenerator.cs]: Thanks a lot for your suggestions. I also think it's reasonable to add some line breaks, and I've fixed it.

Comment (ACGNnsj) [reply Ryujinx.Horizon.Kernel.Generators/Kernel/SyscallGenerator.cs]: > I don't mind LINQ, but those extremely long lines are not good for readability imo. Maybe you can put some line breaks here.

Thanks a lot for your suggestions. I also think it's reasonable to add some line breaks, and I've fixed it.🙂

**Review by gdkchan — APPROVED** [review]
lgtm, thanks.

**Review by TSRBerry — APPROVED** [review]
Lgtm

**Pull request merged**